### PR TITLE
New package: zsh-autosuggestions-0.4.3

### PR DESCRIPTION
**Pull request by vinnyA3**
Added package template for the zsh-autosuggestions plugin

### CODE REVIEW

File: srcpkgs/zsh-autosuggestions/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Don't forget to source zsh-autosuggestions in your .zshrc file:
+
```
Comment (maxice8) [inline srcpkgs/zsh-autosuggestions/INSTALL.msg]: This is not necessary

Comment (vinnyA3) [reply srcpkgs/zsh-autosuggestions/INSTALL.msg]: @maxice8 Should I remove the message, or the whole *INSTALL.md*?

**Comment by maxice8** [comment]
On Sat, Nov 17, 2018 at 05:58:05PM -0800, Vinny wrote:
> vinnyA3 commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,3 @@
> +Don't forget to source zsh-autosuggestions in your .zshrc file:
> +
> 
> @maxice8 Should I remove the message, or the whole *INSTALL.md*?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4647#discussion_r234426871

the whole INSTALL.md

**Comment by vinnyA3** [comment]
All done :) @maxice8

**Comment by maxice8** [comment]
There are 2 commits, please squash them together into one

**Comment by vinnyA3** [comment]
Sorry about that!  I made the squash

File: srcpkgs/zsh-autosuggestions/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+pkgname=zsh-autosuggestions
+version=0.4.3
+revision=1
+noarch=yes
+depends="zsh"
+short_desc="Fish-like autosuggestions for zsh"
+maintainer="vinnyA3 <vincent.aceto@gmail.com>"
+license="MIT"
+homepage="https://github.com/zsh-users/zsh-autosuggestions"
+distfiles="${homepage}/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
```
Comment (maxice8) [inline srcpkgs/zsh-autosuggestions/template]: There is no need to rename the distfiles

Comment (vinnyA3) [reply srcpkgs/zsh-autosuggestions/template]: Fixed distfiles rename

**Comment by vinnyA3** [comment]
Thanks for the review and merge @maxice8 Glad to be part of the community!

**Pull request merged**